app/src/main/java/org/asteroidos/sync/asteroid/IAsteroidDevice.java

```diff
@@ -0,0 +1,47 @@
+package org.asteroidos.sync.asteroid;
+
+import org.asteroidos.sync.connectivity.IConnectivityService;
+import org.asteroidos.sync.connectivity.IServiceCallback;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface IAsteroidDevice {
+    String name = "";
+    String macAddress = "";
+    int batteryPercentage = 0;
+    boolean bonded = false;
+
+    enum ConnectionState {
+        STATUS_CONNECTED,
+        STATUS_CONNECTING,
+        STATUS_DISCONNECTED
+    }
+
+    /**
+     * Can be used to determine if an {@link IAsteroidDevice} is {@link IAsteroidDevice.ConnectionState#STATUS_CONNECTED},
```

Comment:
I find it weird that this is the only comment in this interface.

Also, the name of the method is quite self expanatory so I don't find the comment particularly helpful. I know just as much when reading "getConnectionState" than when reading

```
public ConnectionState</b> getConnectionState() {
     * // determine connection state
     * // return {@link ConnectionState}
 [...]
     * @return Current state of the {@link IAsteroidDevice} connection.
```

which is basically the same thing said 4 times :p